parameters/navigation/global_costmap_gmapping.yaml

```diff
@@ -3,7 +3,7 @@
 global_frame: /map
 robot_base_frame: /amigo/base_link
 rolling_window: false
-track_unknown_space: true
+track_unknown_space: false
```

Comment:
This is quite a change...

Reply:
If he doesn’t we should make a separate one for the restaurant challenge

Reply:
ping @reinzor

Reply:
Rein: lijkt me goed